db/schema.rb

```diff
@@ -10,11 +10,27 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_04_27_124337) do
+ActiveRecord::Schema[7.0].define(version: 2023_10_01_065435) do
```

Comment:
![image](https://github.com/MaMpf-HD/mampf/assets/37160523/4a60762d-b3cb-4098-8f0d-441172313513)

I've just purged all my Docker data, initialized Mampf with the seed data and went to the lecture edit page. For `Lineare Algebra 2, WS 2022/23`, no option was selected. Likewise for any other lecture I have tested. Running `rake db:migrate` did not help either.

More concerning though: when I clicked on "Yes", this change was not reflected and users could still not make their annotations visible for the teacher.

Reply:
In the database, `annotation_status` has the value `nil`. We probably want a default value there.

Reply:
You're right, I should definitely add a default value to the schema.
I'm not entirely sure, but from the description of your problem, I would assume, that you didn't set a value on the medium level: Currently, if no value is set on the medium level, it also deactivates the feature by default, even if a value is set on the lecture level.